ursh/indico_ursh/controllers.py

```diff
@@ -52,8 +56,52 @@ def _process(self):
         return jsonify(url=short_url)
 
 
-class RHDisplayShortenURLPage(RH):
+class RHShortURLPage(RH):
     """Provide a simple page, where users can submit a URL to be shortened"""
 
     def _process(self):
-        return WPShortenURLPage.render_template('url_shortener.html')
+        return WPShortenURLPage.render_template('ursh_shortener_page.html')
+
+
+class RHCustomShortURLPage(RHEventBase):
```

Comment:
this should inherit from the event management RH since only event managers may use it